sofa-ark-parent/core-impl/container/src/main/java/com/alipay/sofa/ark/container/service/classloader/AbstractClasspathClassloader.java

```diff
@@ -270,7 +296,7 @@ public URL getResource(String name) {
      * @return
      */
     protected URL getExportResource(String resourceName) {
-        if (shouldFindExportedResource(resourceName)) {
+        if (!StringUtils.isEmpty(resourceName) && shouldFindExportedResource(resourceName)) {
```

Comment:
There is another method `getExportResources` , it's better to add a default logic on `shouldFindExportedResource`